examples/pbf2d.py

```diff
@@ -46,21 +46,21 @@ def round_up(f, s):
 # corrN = 4.0
 neighbor_radius = h * 1.05
 
-poly6_factor = 315.0 / 64.0 / np.pi
-spiky_grad_factor = -45.0 / np.pi
+poly6_factor = 315.0 / 64.0 / math.pi
+spiky_grad_factor = -45.0 / math.pi
 
-old_positions = ti.Vector(dim, dt=ti.f32)
-positions = ti.Vector(dim, dt=ti.f32)
-velocities = ti.Vector(dim, dt=ti.f32)
+old_positions = ti.Vector.field(dim, float)
+positions = ti.Vector.field(dim, float)
+velocities = ti.Vector.field(dim, float)
 # Once taichi supports clear(), we can get rid of grid_num_particles
```

Comment:
What do you mean by `clear()`?

Reply:
Maybe we need to ask the author of this example...

Reply:
I wanted to use `dynamic` initially, but `dense` seems to work OK. So I think we can remove this comment now.

```suggestion
```